tasks/cleanup-workspace/README.md

```diff
@@ -4,9 +4,10 @@ Tekton task to delete a given directory in a passed workspace.
 
 ## Parameters
 
-| Name | Description | Optional | Default value |
-|------|-------------|----------|---------------|
-| subdirectory | The directory to remove within the workspace | No | - |
+| Name         | Description                                                                                           | Optional | Default value |
+|--------------|-------------------------------------------------------------------------------------------------------|----------|---------------|
+| subdirectory | The directory to remove within the workspace                                                          | No | - |
+| delay        | Time to delay execution. Needed to allow other finally tasks to access workspace before being deleted | Yes | 60s |
```

Comment:
The "s" was removed, so the default value is `60`.

Reply:
fixed